assets/scss/styles.scss

```diff
@@ -126,69 +152,52 @@ h1 {
 }
 
 /* Stats page */
-#stats {
+.stat_data {
+  h2 {
+    @include bold-24;
+  }
   table {
+    margin: 0 50px 50px;
```

Comment:
Tom and I agree that using px based measurements is not accessible. This was based on Karwai's designs at the time. Will be revising.